### ISSUE TITLE
v1.8.3: missing overal val/loss and train/loss in progres bar of Epoch

### ISSUE THREAD

**Issue by LeMinhNgan** [roboflow/rf-detr#1184]
### Search before asking

- [x] I have searched the RF-DETR [issues](https://github.com/roboflow/rf-detr/issues) and found no similar feature requests.

### Description

Things seem good with the v1. 8.3 unless we miss the val/loss; also the train/loss group only show the current batch losses not the epoch-level losses. We read those numbers but cannot get the whole picture of val/loss and train loss of the epoch because those values are calculated for batch-level, not epoch-level. If you can add the losses values when finish an epoch, that information will be informative since it's not always convenient to search them in tensorboard

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!
